backend/bitsurance/bitsurance.go

```diff
@@ -54,7 +54,7 @@ func bitsuranceCheckId(httpClient *http.Client, accountId string) (bool, error)
 	endpoint := apiURL + "accountDetails/" + accountId
 	details := accountDetails{}
 	code, err := util.APIGet(httpClient, endpoint, apiKey, 1024, &details)
-	if err != nil {
+	if err != nil && code != http.StatusNotFound {
```

Comment:
404 is not a real error in this case, as it is the expected response in case an account is not insured.